libsnark/common/data_structures/integer_permutation.hpp

```diff
@@ -31,6 +31,7 @@ class integer_permutation
     integer_permutation(const size_t size = 0);
     integer_permutation(const size_t min_element, const size_t max_element);
 
+    integer_permutation(const integer_permutation &other) = default;
```

Comment:
Why is this one needed?

Reply:
Without this, the compiler gives a warning that an assignment operator has been defined, but a copy-constructor has not.

Reply:
Ah I see, thanks